README.md

```diff
@@ -1,3 +1,50 @@
 # godess-tribes-frontend
 
 This is the repository of Tribes Front-End Project of Team Godess
+
+# Project Summary
```

Comment:
Please use a hierarchy for headings, one page should consist of one top-level heading.